### PR TITLE
LRAC-10725

### PR DESCRIPTION
**Pull request by marcosapmf**
https://issues.liferay.com/browse/LRAC-10725

cc @rachaelkoestartyo

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeGroup.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	protected void doUpgrade() throws Exception {
+		if (!hasColumn("Group_", "modifiedDate")) {
+			alterTableAddColumn("Group_", "modifiedDate", "DATE");
+		}
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				"update Group_ set modifiedDate = ? where modifiedDate is " +
+					"null")) {
+
+			preparedStatement.setDate(1, new Date(System.currentTimeMillis()));
```
Comment (marcellustavares) [inline portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeGroup.java:37]: shouldn't we default to createDate?

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcellustavares** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

File: portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeGroup.java (hunk 1 above)
Comment (rachaelkoestartyo) [reply portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeGroup.java:37]: there's no createDate either 😄

**Pull request closed without merging**